agreementbot/object_manager.go

```diff
@@ -195,14 +195,18 @@ func (m *MMSObjectPolicyManager) UpdatePolicies(org string, updatedPolicies *exc
 		glog.V(5).Infof(mmsLogString(fmt.Sprintf("Starting object policy garbage collection")))
 		for org, serviceMap := range m.orgMap {
 			for service, peList := range serviceMap {
-				for ix, pe := range peList {
+				finalList := make([]MMSObjectPolicyEntry, 0)
```

Comment:
Please add a comment before this line, something like:
// Since we cant modify the policy entry list while iterating over it, collect the entries that we want to keep and add them back into the object policy manager once the iteration is complete.

And same comment in the next section too.